src/quantum/azext_quantum/_help.py

```diff
@@ -18,6 +18,10 @@
       - name: Submit the Q# program from the current folder and wait for the result.
         text: |-
             az quantum execute -g MyResourceGroup -w MyWorkspace -l MyLocation -t MyTarget
+      - name: Submit and wait for a Q# program from the current folder with job and program parameters.
+        text: |-
+            az quantum execute -g MyResourceGroup -w MyWorkspace -l MyLocation -t MyTarget \\
+                --job-params key1=value1 key2=value2 -- --n-qubits 3
```

Comment:
I think there's redundant `--` in the example
```suggestion
                --job-params key1=value1 key2=value2 --n-qubits 3
```

Reply:
@kairu-ms, ah, that's intentional. Because of legacy reasons, we have a `--` separating the CLI parameters from a positional argument that includes all the parameters that get sent to the executable itself being run in Azure Quantum. So the `--` is the existing delimiter to separate CLI extension parameters from the submitted .exe parameters.

This has been there since version 1, however, I'm driving for a redesign in future versions to do something similar to --job-params instead.

Reply:
@ricardo-espinoza 
It's weird to add extra `--` here and it's difficult for users to understand the argument usage.
BTW, could you point us where you handle this extra `--` in code?

Reply:
This information has been provided offline. The usage of `--` is non-standard and I agree we should change it at some point, but it has been there since the first version, and is probably out of the scope of this PR.